x/merkledb/db_test.go

```diff
@@ -675,15 +678,19 @@ func Test_MerkleDB_RandomCases(t *testing.T) {
 	require := require.New(t)
 
 	for i := 150; i < 500; i += 10 {
-		r := rand.New(rand.NewSource(int64(i))) // #nosec G404
+		now := time.Now().UnixNano()
+		t.Logf("seed for iter %v: %v", i, now)
```

Comment:
I feel like it's slightly more clear to use the correct directive rather than `%v` for everything.
```suggestion
		t.Logf("seed for iter %d: %d", i, now)
```